core/capabilities/ccip/configs/solana/chain_writer.go

```diff
@@ -244,70 +244,6 @@ func getExecuteMethodConfig(fromAddress string, offrampProgramAddress string) ch
 				},
 				Optional: true,
 			},
-			{
-				PDALookups: &chainwriter.PDALookups{
-					Name: "ReceiverAssociatedTokenAccount",
-					PublicKey: chainwriter.Lookup{
-						AccountConstant: &chainwriter.AccountConstant{
-							Address: solana.SPLAssociatedTokenAccountProgramID.String(),
-						},
-					},
-					Seeds: []chainwriter.Seed{
-						{Dynamic: chainwriter.Lookup{AccountLookup: &chainwriter.AccountLookup{Location: "Info.AbstractReports.Messages.Receiver"}}},
-						// Token Program stored in PoolLookupTable
-						{Dynamic: chainwriter.Lookup{
-							AccountsFromLookupTable: &chainwriter.AccountsFromLookupTable{
-								LookupTableName: "PoolLookupTable",
-								IncludeIndexes:  []int{6},
-							},
-						}},
-						{Dynamic: chainwriter.Lookup{AccountLookup: &chainwriter.AccountLookup{Location: destTokenAddress}}},
-					},
-					IsSigner:   false,
-					IsWritable: true,
-				},
-				Optional: true,
-			},
-			{
-				PDALookups: &chainwriter.PDALookups{
-					Name:      "PerChainTokenConfig",
-					PublicKey: getFeeQuoterProgramAccount(offrampProgramAddress),
-					Seeds: []chainwriter.Seed{
-						{Static: []byte("per_chain_per_token_config")},
-						{Dynamic: chainwriter.Lookup{AccountLookup: &chainwriter.AccountLookup{Location: sourceChainSelectorPath}}},
-						{Dynamic: chainwriter.Lookup{AccountLookup: &chainwriter.AccountLookup{Location: destTokenAddress}}},
-					},
-					IsSigner:   false,
-					IsWritable: false,
-				},
-				Optional: true,
-			},
-			{
-				PDALookups: &chainwriter.PDALookups{
-					Name: "PoolChainConfig",
-					PublicKey: chainwriter.Lookup{
-						AccountsFromLookupTable: &chainwriter.AccountsFromLookupTable{
-							LookupTableName: "PoolLookupTable",
-							IncludeIndexes:  []int{2},
-						},
-					},
-					Seeds: []chainwriter.Seed{
-						{Static: []byte("ccip_tokenpool_chainconfig")},
-						{Dynamic: chainwriter.Lookup{AccountLookup: &chainwriter.AccountLookup{Location: sourceChainSelectorPath}}},
-						{Dynamic: chainwriter.Lookup{AccountLookup: &chainwriter.AccountLookup{Location: destTokenAddress}}},
-					},
-					IsSigner:   false,
-					IsWritable: true,
-				},
-				Optional: true,
-			},
-			{
-				AccountsFromLookupTable: &chainwriter.AccountsFromLookupTable{
-					LookupTableName: "PoolLookupTable",
-					IncludeIndexes:  []int{},
-				},
-				Optional: true,
-			},
```

Comment:
Could we add a comment here mentioning that these accounts for token transfers are added during the CCIPExecute args transform? Something like:
```
// user token account, token billing config, pool chain config, and pool lookup table accounts are added to the accounts list in the CCIPExecute args transform for each token transfer
```